articles/securing-ai-systems-from-adversaries/index.md

```diff
@@ -0,0 +1,138 @@
+
+ ![matrix](/engineering-education/securing-ai-systems-from-adversaries/matrix.jpg)
+
+Adversarial ML Threat Matrix [Source](https://www.helpnetsecurity.com/2020/10/27/attacks-machine-learning-systems/)
+
+#### Model Duplicating Techniques
+
+Under this criterion, the approaches taken are used to steal or duplicate the targeted models. This may involve stealing/duplicating the model itself or getting access to the training data of the model. The training data in question may be sensitive as well as highly valuable. For example, private client financial information, confidential military data, and patient information. As a result, an attack of such a nature may end up leaking the data. Data also makes up a telling part of intellectual property. Leakage of such information will result in hefty consequences to an organization.
+
+##### Model Replication
+
+An example of an attack is model replication. This attack can involve the exploitation of a public-facing API to reconstruct a model. Public APIs that may prove to be worthy targets of attacks can be found in cloud-based machine learning services of many companies. These companies provide and run training algorithms that use datasets uploaded by users. The queries involved are often prediction queries. The interaction between users and the algorithms is handled by user-friendly Web APIs. If these models are successfully monetized by the owners, an attacker may be motivated to attack to bypass query charges. This undermines the business model of the model owner. An attacker may look to violate training-data privacy by leaking sensitive training data. The public API is probed to gradually refine a model owned by the attacker.
+
+##### Defense
+
+An effective way to defend AI from model duplicating techniques is the use of a privacy framework named Private Aggregation of Teacher Ensembles (PATE). PATE is motivated by the concerns about the privacy of sensitive data that is used to train several machine learning models. These models need to be prevented from revealing confidential details of the sensitive data.
+
+PATE works on the principle that involves training several models on disjoint data. These models are referred to as “teacher” models. If these models agree on input, no confidential details from their training set are leaked. The models agree since they all reach the same conclusion. For privacy to be achieved, the outputs of the aforementioned models need to be in consensus.
```

Comment:
What do you mean by these models agree on input? It's not clear how they reach the same conclusion. If the models are trained on dis-joint data, then how will they reach the same conclusion? Is it happening batch-wise?